src/modules/dracut/main.py

```diff
@@ -53,4 +53,4 @@ def run():
 
     if return_code != 0:
         return (_("Failed to run dracut on the target"),
-                _("The exit code was {}").format(return_code))
+                _(f"The exit code was {return_code}"))
```

Comment:
Not clear why the `f` here, causes invalid python code on my end.

Reply:
That is a python "f string".  As of Calamares 3.3 we are using those.  What version of Python are using?

As a side note, when I was testing that, I noticed an exception handling bug which I am fixing now.

Reply:
Should be working fully now